brjs-core/src/main/java/org/bladerunnerjs/api/plugin/AssetPlugin.java

```diff
@@ -1,16 +1,23 @@
 package org.bladerunnerjs.api.plugin;
 
+import java.util.List;
+
 import org.bladerunnerjs.api.Asset;
-import org.bladerunnerjs.api.AssetLocation;
-import org.bladerunnerjs.api.LinkedAsset;
-import org.bladerunnerjs.api.SourceModule;
 import org.bladerunnerjs.api.memoization.MemoizedFile;
-import org.bladerunnerjs.model.AssetFileInstantationException;
+import org.bladerunnerjs.model.AssetContainer;
+
 
-/**
- * Asset plug-ins allow new implementations of {@link SourceModule}, {@link LinkedAsset} &amp; {@link Asset} to be supported within the model.
- */
-public interface AssetPlugin extends OrderedPlugin {
-	boolean canHandleAsset(MemoizedFile assetFile, AssetLocation assetLocation);
-	Asset createAsset(MemoizedFile assetFile, AssetLocation assetLocation) throws AssetFileInstantationException;
+public interface AssetPlugin extends Plugin
+{
+	/**
+	 * Get the priority for this plugin. Lower priorities correspond to earlier execution.
+	 * The default priority for plugins is 10.
+	 * Any plugin that can detirmine with 100% confidence it can handle a given resource should define a lower priority so it is executed first and can better influence the discovery of assets.
+	 * 
+	 * @return the priority for this plugin. Lower priorities correspond to earlier execution.
+	 */
+	int priority();
```

Comment:
I think this is worse than what we had before as it will no longer always be possible for plug-ins to ensure that they will appear in the correct order relative to other plug-ins, depending on the `priority` values those other plug-ins have chosen for themselves.

Instead, it probably makes more sense to have the plug-in order determined globally within `brjs.conf`, for example:

```
pluginOrder: Plugin1, Plugin2, *, PluginN
```